<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegFrame</class>
 <widget class="QFrame" name="RegFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="placeholderText">
    <string>Bio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Re:password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SUBMIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="switchToLog">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK TO LOGIN</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
